usr/src/cmd/zfs/zfs_main.c

```diff
@@ -3892,7 +3896,7 @@ zfs_do_send(int argc, char **argv)
 static int
 zfs_do_receive(int argc, char **argv)
 {
-	int c, err;
+	int c, err = 0;
```

Comment:
it is probably gcc bug, but we have warning:
may be used uninitialized in this function [-Wuninitialized]